bin/ch_core.c

```diff
@@ -68,26 +68,29 @@ struct {
 
 /** Function prototypes (private) **/
 
-void bind_mount(char *src, char *dst, char *newroot,
-                enum bind_dep dep, unsigned long flags);
-void bind_mounts(struct bind *binds, char *newroot,
+void bind_mount(char const * const src, char const * const dst,
+                char const * const newroot, enum bind_dep dep,
+                unsigned long flags);
+void bind_mounts(struct bind const * const binds, char const * const newroot,
```

Comment:
In the meeting, we decided that const should only be used to ensure that functions can't change variables that are passed by reference. That would mean removing `const * const` since the second one (?) only affects what's inside the function.